src/MySqlConnector/MySqlClient/MySqlBulkLoader.cs

```diff
@@ -196,7 +203,24 @@ private string BuildSqlCommand()
             }
         }
 
-        internal const string StreamPrefix = ":STREAM:";
+		private Stream CreateFileStream(string fileName)
+		{
+			try
+			{
+				return new FileStream(fileName, FileMode.Open);
```

Comment:
Use `File.OpenRead` because it specifies good defaults for `FileAccess` and `FileShare`.